### ISSUE TITLE
Module version mismatch when using NAPI enabled Node-ChakraCore 7.x

### ISSUE THREAD

**Issue by aruneshchandra** [nodejs/abi-stable-node#15]
```
C:\git\demo>..\NAPI-Node-Chakra\node.exe app.js
Error: Module version mismatch. Expected 48, got -1.
   at bindings (C:\git\demo\node_modules\bindings\bindings.js:83:9)
   at Anonymous function (C:\git\demo\node_modules\leveldown\leveldown.js:4:7)
   at Module.prototype._compile (module.js:541:3)
   at Module._extensions[.js] (module.js:550:3)
   at Module.prototype.load (module.js:458:3)
   at tryModuleLoad (module.js:417:5)
   at Module._load (module.js:409:3)
   at Module.prototype.require (module.js:468:3)
   at require (internal/module.js:20:7)
   at Anonymous function (C:\git\demo\app.js:1:63)
```

**Comment by sampsongao** [comment]
I think we just need to change the check to allow version -1 in DLOpen in the src/node.cc.

**Comment by boingoing** [comment]
Fixed with e8de3a7b8394e08b2d81d889a10ca0577f469f20
